src/components/radio-group/radio-group.ts

```diff
@@ -26,6 +26,26 @@ export default class SlRadioGroup extends LitElement {
   /** Shows the fieldset and legend that surrounds the radio group. */
   @property({ type: Boolean, attribute: 'fieldset' }) fieldset = false;
 
+  /** Indicates that a selection is required. */
+  @property({ type: Boolean, reflect: true }) required = false;
+
+  constructor() {
+    super();
+    this.addEventListener('sl-change', this.syncRadioButtons);
+  }
+
+  disconnectedCallback() {
+    this.removeEventListener('sl-change', this.syncRadioButtons);
+  }
+
+  syncRadioButtons(ev: CustomEvent) {
```

Comment:
```suggestion
  syncRadioButtons(event: CustomEvent) {
```

Nitpicking here, but `event` is used pretty consistently.